srcpkgs/gamemode/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'gamemode'
+pkgname=gamemode
+version=1.5.1
+revision=1
+build_style=meson
+configure_args="-Dwith-systemd=false -Dwith-dbus-service-dir=/usr/share/dbus-1/services --prefix=/usr --buildtype debugoptimized"
+hostmakedepends="git cmake meson ninja pkg-config"
+makedepends="inih-devel dbus-devel elogind-devel"
+short_desc="Daemon/lib combo for optimisations running games"
```

Comment:
I think the projects description "Optimise Linux system performance on demand" is better.